### ISSUE TITLE
Assets folder into a single file

### ISSUE THREAD

**Issue by vzenix** [originjs/vite-plugin-federation#489]
### Is your feature request related to a problem? Please describe.

When I use various aplications with it generate to much js in assets folders, in some device the load of application is slow for the number of js in assets folder combinated between apps

### Describe the solution you'd like

I want build a single file between the current multiple file in assets folder

**Comment by nguyenbatranvan** [comment]
Please check my plugin
[Vite share assets](https://www.npmjs.com/package/vite-multiple-assets)

**Comment by RaikoM** [comment]
Hi, did you find a solution?

**Comment by vzenix** [comment]
> Hi, did you find a solution?

No, I have pending search how to change cache in http header for files with hash 100% and no cache for statics names

At the same time we search now how do laizy load

**Comment by Zhang-Liangjie** [comment]
Hi, did you find a solution?

**Comment by vzenix** [comment]
I left the project with this solution

But the only solution I found was to work the cache on the provider (apache, iis,...)
